packages/selinux-policy/rules.cil

```diff
@@ -53,13 +53,13 @@
 (allow init_t runtime_t (processes (transform)))
 (allow runtime_t runtime_exec_t (file (entrypoint)))
 
-; `runc` starts container processes as "container_t" by default, but
-; it can use other "container" subject labels like "super_t".
+; `runc` starts container processes as "control_t" by default, but
+; it can use other "container" subject labels like "container_t".
```

Comment:
I think it could be worth expanding this a little bit, because (naively read) it seems to imply that privileged is the default, since here it says control_t is default, and the docs above say "control_t (the default for privileged containers)".  This says "can use other [..] labels" but (again, naively) it's not clear when or why it would.  Offline you mentioned a process that automatically applies process labels, including making the default label container_t, down from control_t -- could there be a few words about that here, perhaps?  I'm totally willing to accept that this is my naivete on SELinux and that it's clear enough if you have context.